staging/src/k8s.io/client-go/scale/interfaces.go

```diff
@@ -22,10 +22,14 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-// ScalesGetter can produce a ScaleInterface
-// for a particular namespace.
+// ScalesGetter can produce a ScaleInterface.
 type ScalesGetter interface {
+	// Scales produces a ScaleInterface for a particular namespace.
 	Scales(namespace string) ScaleInterface
```

Comment:
does this func work if the argument is empty? I.e. the funcs below are just convenience funcs?

Reply:
No it doesn't. It returns an error.